packages/@expo/metro-config/src/transform-worker/asset-transformer.ts

```diff
@@ -132,6 +133,7 @@ export async function transform(
 
       return {
         ast: {
+          comments: null,
```

Comment:
Should this one be included?

Reply:
yes, upgrading react-native-reanimated bumped the patch version of babel-parser and typescript changed a bit, we now need to explicitly set comments as null